app/views/overrides/activation_keys/_host_environment_select.html.erb

```diff
@@ -15,26 +15,33 @@
 <% cs_select_id =  using_hostgroups_page? ? :content_source_id : :content_source_id %>
 <% cs_select_name =  using_hostgroups_page? ? 'hostgroup[content_source_id]' : 'host[content_facet_attributes][content_source_id]' %>
 <% cs_select_attr = using_hostgroups_page? ? 'content_source' : 'content_facet.content_source' %>
+<% content_source_inherited = true if content_source_inherited? %>
```

Comment:
Now that we've moved the logic to the helper, I don't see a reason to assign these to variables anymore..

Reply:
yeah good point :)